submissions/yevheniiii/html_css_popup/index.html

```diff
@@ -0,0 +1,147 @@
+<!DOCTYPE html>
+<html lang="uk">
+
+<head>
+    <title>Kottans Pop-up</title>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="css/main.css" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+</head>
+
+<body>
+    <nav>
+        <section class="menu">
```

Comment:
Good practice to use list structure for menu.
Use `ul > li` istead of `section > section`.
Read more about HTML tag `<section>` pay attention to its semantics.

Reply:
Thank you! I changed this.